runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -327,6 +327,51 @@ void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::Tr
    treetop->insertAfter(TR::TreeTop::create(comp(), TR::Node::create(node, TR::treetop, 1, newCallNode)));
    }
 
+void J9::RecognizedCallTransformer::process_jdk_internal_util_ArraysSupport_vectorizedMismatch(TR::TreeTop* treetop, TR::Node* node)
+   {
+   TR::Node* a = node->getChild(0);
+   TR::Node* aOffset = node->getChild(1);
+   TR::Node* b = node->getChild(2);
+   TR::Node* bOffset = node->getChild(3);
+   TR::Node* length = node->getChild(4);
+   TR::Node* log2ArrayIndexScale = node->getChild(5);
+
+   TR::Node* lengthInBytes = TR::Node::create(node, TR::ishl, 2, length, log2ArrayIndexScale);
+   TR::Node* mask = TR::Node::create(node, TR::ior, 2,
+      TR::Node::create(node, TR::ishl, 2,
+         log2ArrayIndexScale,
+         TR::Node::iconst(node, 1)),
+      TR::Node::iconst(node, 3));
+   TR::Node* n = TR::Node::create(node, TR::iand, 2,
```

Comment:
What is the use of `n` ?  I understand the reason behind using single letter variable name for the child of the call node as that is how java method is declared. For other case, please use names closer to how they are used.